hermes/src/api/ws.rs

```diff
@@ -283,13 +357,12 @@ impl Subscriber {
     }
 }
 
-pub async fn notify_updates(ws_state: Arc<WsState>) {
-    let closed_subscribers: Vec<Option<SubscriberId>> = join_all(
-        ws_state
-            .subscribers
-            .iter_mut()
-            .map(|subscriber| async move {
-                match subscriber.send(()).await {
+pub async fn notify_updates(ws_state: Arc<WsState>, event: AggregationEvent) {
+    let closed_subscribers: Vec<Option<SubscriberId>> =
+        join_all(ws_state.subscribers.iter_mut().map(|subscriber| {
+            let event = event.clone();
+            async move {
+                match subscriber.send(event.clone()).await {
```

Comment:
Do you need this `.clone()` if we already cloned above? Should only consume the copy.